src/js/pages/ReadyLight.jsx

```diff
@@ -66,6 +70,42 @@ class ReadyLight extends Component {
     window.scrollTo(0, 0);
   }
 
+  // Terry - Changes made for WV-1448 adding landing datalayers
+  componentDidUpdate () {
+    const { dataLayerFired } = this.state;
+    // Terry - only fire datalayer when voter data is ready
```

Comment:
No need to add your name for routine changes, we can use git blame to see who made them.  What you are explaining could help someone, but if everyone added their name for every change the code would get very messy.  It is better to remove all the comments with your name in this case.